src/inc/msquic.h

```diff
@@ -241,6 +242,12 @@ typedef struct QUIC_CERTIFICATE_FILE_PROTECTED {
     const char *PrivateKeyPassword;
 } QUIC_CERTIFICATE_FILE_PROTECTED;
 
+typedef struct QUIC_CERTIFICATE_PKCS12 {
+    const uint8_t *Asn1Blob;
+    int Asn1BlobLength;
```

Comment:
No `int` please. There is no reason for this length to be negative. `uint32_t` instead.

Reply:
updated. Sorry for the turn. I try to follow code around but the conventions I normally use are different and it is sometimes hard for me to spot the differences.